Mathlib/Analysis/SpecialFunctions/Pow/Deriv.lean

```diff
@@ -601,6 +609,20 @@ theorem deriv_rpow_const (hf : DifferentiableAt ℝ f x) (hx : f x ≠ 0 ∨ 1 
   (hf.hasDerivAt.rpow_const hx).deriv
 #align deriv_rpow_const deriv_rpow_const
 
+lemma isBigO_deriv_rpow_const_atTop (p : ℝ) :
+    deriv (fun (x:ℝ) => x ^ p) =O[atTop] fun x => x ^ (p-1) := by
+  by_cases hp : p = 0
```

Comment:
`rcases eq_or_ne p 0 with rfl | hp`? Also, for `p ≠ 0` you can have `IsTheta`.

Reply:
I did this and extracted the `p ≠ 0` case as a separate lemma where we get `IsTheta`.